appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -4345,6 +4368,28 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   String widthParams();
 
   //Events
+
+  @DefaultMessage("ReachLeftEnd")
```

Comment:
I'd like to think about this w.r.t. RTL support. In iOS, the terms used are leading and trailing edges where leading is left in LTR languages and right in RTL languages, and vice versa for trailing. We should look at how these events behave on devices configured with RTL languages. Do the semantics end up backward?